src/Features/VisualBasic/Portable/Completion/CompletionProviders/NamedParameterCompletionProvider.vb

```diff
@@ -43,12 +43,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
             End If
 
             If token.Kind = SyntaxKind.CommaToken Then
-                For Each n In argumentList.Arguments.GetWithSeparators()
-                    If n.IsNode AndAlso DirectCast(n.AsNode(), ArgumentSyntax).IsNamed Then
-                        context.IsExclusive = True
-                        Exit For
-                    End If
-                Next
+                Dim languageVersion = DirectCast(document.Project.ParseOptions, VisualBasicParseOptions).LanguageVersion
+                If languageVersion < LanguageVersion.VisualBasic15_5 Then
+                    For Each n In argumentList.Arguments.GetWithSeparators()
```

Comment:
i dislike the asymmetry here with C#.  Can we add a similar token...IsMandatoryYaddaYadda for VB as well so that our code tends to stay better in sync?